libs/blocks/quiz/quiz.js

```diff
@@ -258,6 +258,10 @@ const App = ({
     if (fragmentURL) {
       loadFragments(fragmentURL);
     }
+    const iconBg = getStringValue('icon-background-color');
```

Comment:
Could this be done after page load?

Reply:
I've placed it here so authors have the option of different backgrounds per question - not sure if there's a use-case for that or not.